src/plugins/dashboard/public/dashboard_container/component/grid/dashboard_grid_item.tsx

```diff
@@ -69,12 +72,30 @@ const Item = React.forwardRef<HTMLDivElement, Props>(
         if (highlightPanelId === id) {
           container.highlightPanel(ref.current);
         }
+
+        ref.current.querySelectorAll('*').forEach((e) => {
+          if (focusPanelId && focusPanelId !== id) {
+            // remove blurred panels and nested elements from tab order
+            e.setAttribute('tabindex', '-1');
+          } else {
+            // restore tab order
+            e.removeAttribute('tabindex');
+          }
+        });
       }
-    }, [id, container, scrollToPanelId, highlightPanelId, ref]);
+    }, [id, container, scrollToPanelId, highlightPanelId, ref, focusPanelId]);
+
+    const focusStyles =
+      focusPanelId && focusPanelId !== id
+        ? css`
+            pointer-events: none;
+            opacity: 0.25;
+          `
+        : css``;
 
     return (
       <div
-        style={{ ...style, zIndex: focusedPanelId === id ? 2 : 'auto' }}
```

Comment:
@ThomThomson Before `style` was destructured from props to override z-index on `focusedPanelId` which wasn't being used, and it doesn't seem necessary to change the z-index, so I removed the z-index style and removed the `style` previously destructured on line 46.